docs/UserGuide.md

```diff
@@ -90,18 +90,34 @@ gradlew run -Dargs="-view output_path/reposense-report"
 ```
 `-Dargs="..."` uses the same argument format as mentioned above.
 
-## CSV Config File
-The `CSV Config File` contains the list of repositories, and the corresponding target authors to track contribution of.
+## CSV Config Files
+The `repo-config.csv` contains the location and the branch of repositories to analyze.
 [repo-config.csv](../repo-config.csv) is an example of a configuration file setup. It should contain the following columns:
 
+Column Name | Explanation
+----------- | -----------
+Repository's Location | The `GitHub URL` or `Disk Path` to the git repository
+Branch | The branch to analyse in the target repository
+[Optional] Ignore Global List | The list of file path globs to ignore during analysis for each author. More details on the Java glob standard [here](https://javapapers.com/java/glob-with-java-nio/)
+
+The [Optional]`author-config.csv` contains the list of repositories, and the corresponding target authors to track contribution of.
```

Comment:
#### Author configuration file [Optional]
to configure the list of authors to analyse and the options.
[author-config.csv](../author-config.csv) is an example of a configuration file setup. It should contain the following columns: